### DISCUSSION TITLE
Despite the filter set on the start page, the total stock is always displayed

### DISCUSSION THREAD

**Discussion by Krabtus** [ghostfolio/ghostfolio#3623] [Q&A]
Hello everyone,

I have a problem with the filter function on the Ghostfolio homepage. No matter which account I select in the filter, it always shows me the total balance of all accounts. It seems that the filter is not applied correctly.

**Steps to reproduce**

1. open the home page of Ghostfolio.
2. select a specific account in the filter
3. observe that the total balance is displayed instead of only the data of the selected account.

**Expected behavior**
After selecting a specific account in the filter, only the balance of that account should be displayed.

**Actual behavior**
The total balance is displayed regardless of which account is selected in the filter.

Has anyone experienced a similar issue or have an idea what could be causing this? Are there any known bugs or configuration steps that I may have overlooked?

Many thanks in advance!

**Comment by tvirus97** [comment]
I have the same problem on 2.105.0. The problem is not just for accounts filter but for all filters.

**Reply by Krabtus** [reply]
Yes, unfortunately it affects all filters on the Ghostfolio homepage except the 'Date Range'. There doesn't seem to be a solution to this yet. Maybe @dtslvr can say more about this.

**Comment by Krabtus** [comment]
I would like to point out once again that the filter variables "Accounts", "Tags" and "Asset Classes" in Ghostfolio still do not work for the information on the start page. This problem has already been reported several times on GitHub, but so far there seems to be no official statement or solution.

Is there possibly an update or a planned solution in the near future? It would be helpful to know if this issue is known and actively being worked on.

Thanks in advance for any feedback!
